docs/tutorials/acme/quick-start/index.rst

```diff
@@ -494,6 +501,15 @@ Create this definition locally and update the email address to your own. This
 email required by Let's Encrypt and used to notify you of certificate
 expirations and updates.
 
+.. note::
+
+    If you're using this document as a guide to configure cert-manager for your
```

Comment:
This is very similar to line `465` just above which also mentions the `ClusterIssuer` for all ingresses. Perhaps you should merge this text into that paragraph?

Reply:
Yep I think this should clarify the Issuer/ClusterIssuer distinction, perhaps by merging into the other paragraph.

There's also a part of the documentation that explains this too which can be linked to: https://docs.cert-manager.io/en/latest/tasks/issuers/index.html#difference-between-issuers-and-clusterissuers

Reply:
Thank you for the quick feedback! Please let me know if the changes fully address your concerns.